marklogic-data-hub-central/ui-custom/package.json

```diff
@@ -24,10 +25,12 @@
     "jsdoc": "^3.6.7",
     "lodash": "^4.17.21",
     "luxon": "^2.3.0",
+    "moment": "^2.29.2",
```

Comment:
@AkshayBajajML - If we have to import a new date time library then let's avoid moment js, since it is a legacy library with few security vulnerabilities. Instead, let's try using dayjs here, since that has replaced moment js in HC as well.
Its API is more or less similar to moment js, so it can be as simple as replacing "moment" with "dayjs" keyword in most cases, except where you try to mutate an existing date variable, since dayjs is immutable as apposed to moment. Hope it helps.
@wooldridge - Just FYI. Hope thats alright ?

Reply:
@xnikhil08, yes, that sounds good. We should get rid of moment.js.

Reply:
@xnikhil08 Thanks for pointing that out. I have replaced it with dayJs and it works fine.